subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/Kconfig

```diff
@@ -115,6 +115,21 @@ config FS_MGMT_HASH_SHA256
 	help
 	  Enable SHA256 hash support for mcumgr.
 
+config FS_MGMT_CHECKSUM_HASH_SUPPORTED_CMD
```

Comment:
Use the `CONFIG_MCUMGR_GRP_FS_CHECKSUM_HASH_SUPPORTED_CMD`, to keep the new Kconfig option format for MCUmgr, as You have already did here https://github.com/zephyrproject-rtos/zephyr/pull/50578/files#diff-834286adf361cd54c1b8c09197543580009a48bd6988bc78240a2487ba731f0eR91

Reply:
Fixed